pkg/volume/rbd/rbd_util.go

```diff
@@ -271,16 +270,18 @@ func makeVDPDNameInternal(host volume.VolumeHost, pool string, image string) str
 	return filepath.Join(host.GetVolumeDevicePluginDir(rbdPluginName), pool+"-image-"+image)
 }
 
-// RBDUtil implements diskManager interface.
-type RBDUtil struct{}
+// Util implements diskManager interface.
+type Util struct{}
```

Comment:
Does `Util` need to be public? It seems to be used only in this package.

Reply:
Oops, I only noticed the function name but didn't notice that it is only used in this package. It seems like this function doesn't need to be public, I will modify it, thank you.

Reply:
Updated. PTAL. Thanks.